rust/src/transformations/polars/make_col/mod.rs

```diff
@@ -0,0 +1,191 @@
+use num::One;
+use opendp_derive::bootstrap;
+use polars::prelude::*;
+use std::collections::BTreeSet;
+
+use crate::core::{Function, MetricSpace, StabilityMap, Transformation};
+use crate::domains::{Context, ExprDomain, ExprMetric, LazyFrameDomain};
+use crate::error::*;
+use crate::traits::DistanceConstant;
+
+#[bootstrap(ffi = false)]
+/// Make a Transformation that returns a col(<column_name>) expression for a Lazy Frame.
```

Comment:
Use backticks instead of <> because <> tend to cause formatting issues downstream, and backticks trigger nice rendering in docs.